packages/appeals-service-api/src/services/object-store/index.js

```diff
@@ -0,0 +1,15 @@
+const config = require('../../configuration/config');
+
+const blobMetaGetter = (initContainerClient) => {
```

Comment:
Sorry, have been looking into a live issue
Only other thing to say really is that document api already provides this functionality from the looks of it:
e.g. locally this returns all document metadata for a questionnaire response
http://localhost:3001/api/v1/has-questionnaire:APP%2FQ9999%2FW%2F22%2F1234567

Reply:
Yeah I discussed this with Ben and we decided to access the blob store directly here because we have the locations of the documents in the blob store. I would have moved the code behind that doc serv url to common as well but it's driven by appeal id.